htlcswitch/link_test.go

```diff
@@ -4355,12 +4299,14 @@ func (h *persistentLinkHarness) restartLink(
 		DecodeHopIterators: decoder.DecodeHopIterators,
 		ExtractErrorEncrypter: func(*btcec.PublicKey) (
 			hop.ErrorEncrypter, lnwire.FailCode) {
+
 			return obfuscator, lnwire.CodeNone
 		},
 		FetchLastChannelUpdate: mockGetChanUpdateMessage,
 		PreimageCache:          pCache,
 		OnChannelFailure: func(lnwire.ChannelID,
 			lnwire.ShortChannelID, LinkFailureError) {
+
 		},
```

Comment:
@yyforyongyu There seems to be a false-positive in the linter
```
htlcswitch/link_test.go:4308: multi-line statement should be followed by a newline (whitespace)
			lnwire.ShortChannelID, LinkFailureError) {
```

Reply:
Yeah, unfortunately the linter fails on the rule if the function body is empty. Can you add a `// nolint` comment to line 4308 please? That should silence it.